src/main/java/ip/main/Duke.java

```diff
@@ -0,0 +1,60 @@
+package ip.main;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Contains the entry point of the chatbot program
+ */
+public class Duke {
+    private static Storage storage;
+    private static Ui ui = new Ui();
+    private static Parser parser = new Parser();
+    private static TaskList tasks = new TaskList(ui);
+
+    /**
+     * The entry point of the application.
```

Comment:
Header comment format doesn't follow our guidelines (even though it is a comment to main method, try to follow the format)